content/python/concepts/file-methods/terms/remove/remove.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.remove()'
+Description: 'Allows the user to delete a file.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Files'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The Python `.remove()` file method allows the user to delete a file if it exists in the system or computer. This method is provided by the `os` module which must be imported beforehand. [`.unlink()`](https://www.codecademy.com/resources/docs/python/file-methods/unlink) is an alternative to delete a file.
+
+**Note**: A `FileNotFoundError` is raised if the file is not found or isn't in the current directory.
+
+## Syntax
+
+```pseudo
+os.remove("file_name.file_type")
+
+os.remove("file_path")
+```
+
+- A file string argument that is passed into `.remove()` will be successfully deleted if it exists in the current directory.
+- A file's path can also be passed into `.remove()` if it's outside the current directory.
+
+Entire folders or directories can be removed using the [`.rmdir()`](https://www.codecademy.com/resources/docs/python/file-methods/rmdir) method.
+
+## Example
+
+Use `.remove()` to delete the **my_file.txt** file:
+
+```python
```

Comment:
```suggestion
```py
```

If probably doesn't make a difference, but just to keep in line with [our content standards for regular snippets](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#code-blocks), could we change all instances of \```python to \```py throughout these files?

Reply:
Not at all, I appreciate this! Thanks @Dusch4593